README.md

```diff
@@ -1,12 +1,12 @@
 # concordion-status-info-extension
 
-This [Concordion](www.concordion.org) extension displays a Note and a Reason in the corresponding specification/markdown, when the annotation expectedToFail is used.
+This [Concordion](www.concordion.org) extension displays a Note and a Reason in the corresponding specification, when the annotations ignored, unimplemented or expectedToFail are used.
```

Comment:
```suggestion
This [Concordion](www.concordion.org) extension displays a Note and a Reason in the output, when the `status`  of specific examples in a specification is [set as](https://concordion.org/instrumenting/java/markdown/#implementation-status) `Ignored`, `Unimplemented` or `ExpectedToFail`.
```
The word `annotation` makes me think this is for the `@ExpectedToFail` etc annotations in Java code. I assume it's not applicable to those, but is only when specific examples in the spec have the `c:status` attribute set?

Reply:
Should we allow people to set this on the `@ExpectedToFail` annotation too?

Reply:
Maybe in the next release?...